code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -339,3 +339,68 @@
 			..()
 	else
 		..()
+
+/obj/item/weapon/pocket_mirror //shamelessly copypasted from [mirror.dm]
+	name = "pocket mirror"
+	desc = "Mirror mirror on the wall, who's the most robust of them all? Touching the mirror will bring out Nanotrasen's state of the art hair modification system."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "pocket_mirror"
+	flags = FPRINT
+	w_class = W_CLASS_TINY
+
+	var/shattered = 0
+
+/obj/item/weapon/pocket_mirror/attack_self(mob/user as mob)
+	if (shattered)
+		return
+
+	if (ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if (isvampire(H))
+			if (!(VAMP_MATURE in H.mind.vampire.powers))
+				to_chat(H, "<span class='notice'>You don't see anything.</span>")
+				return
+
+		if (user.hallucinating())
+			switch(rand(1,100))
+				if (1 to 20)
+					to_chat(H, "<span class='sinister'>You look like [pick("a monster","a goliath","a catbeast","a ghost","a chicken","the mailman","a demon")]! Your heart skips a beat.</span>")
+					H.Knockdown(4)
+					return
+				if (21 to 40)
+					to_chat(H, "<span class='sinister'>There's [pick("somebody","a monster","a little girl","a zombie","a ghost","a catbeast","a demon")] standing behind you!</span>")
+					H.emote("scream",,, 1)
```

Comment:
you can do ```H.emote("scream", auto=1)``` if you really want to do that, just be aware if someone renames the argument in the definition, that it will break.